### ISSUE TITLE
Bug: status bar don't have a bottom padding or space between prompt

### ISSUE THREAD

**Issue by Nivethan-Ar** [dracula/tmux#201]
### Describe the bug
i have a simple tmux configuration. after adding dracula/tmux and set the status bar position at the top i realized there is no bottom padding/space.  

following is my tmux.config file
```
# set tmux prefix as ctrl + space
set -g prefix C-Space

# vim like bindings for navigation
setw -g mode-keys vi
bind-key h select-pane -L
bind-key j select-pane -D
bind-key k select-pane -U
bind-key l select-pane -R

# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'alexghergh/nvim-tmux-navigation'
set -g @plugin 'dracula/tmux'

# dracula setup
set -g @dracula-show-powerline true
set -g @dracula-show-flags true
set -g @dracula-show-left-icon session
set -g status-position top
set -g @dracula-left-icon-padding 20 

set -g @dracula-plugins "time ram-usage"

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'

```

### To Reproduce
1.install dracula/tmux plugin
2. set the status bar position to top

### Expected behavior
i want to add a space on the bottom of the status bar through configuration.

### Screenshots
![image](https://user-images.githubusercontent.com/63112138/232182220-ab4ded0d-041c-4e32-a843-017c9f3a5fe8.png)

### System
 - OS: Ubuntu 22.04
- Tmux Version: 3.2a

**Comment by ethancedwards8** [comment]
Hm, this is an interesting issue. I wasn't even aware that the status bar could be moved to the top of the screen. I cannot immediately find anything in the docs to adjust padding around the bars. [One page](https://stackoverflow.com/questions/39714811/annoying-padding-between-tmux-bar-and-bottom-of-terminal-window-when-maximised) I found says that it can be adjusted by changing font size, but I am not sure I buy that. I'll keep looking in the docs.

**Comment by Nivethan-Ar** [comment]
> Hm, this is an interesting issue. I wasn't even aware that the status bar could be moved to the top of the screen. I cannot immediately find anything in the docs to adjust padding around the bars. [One page](https://stackoverflow.com/questions/39714811/annoying-padding-between-tmux-bar-and-bottom-of-terminal-window-when-maximised) I found says that it can be adjusted by changing font size, but I am not sure I buy that. I'll keep looking in the docs.

thanks for the reply @ethancedwards8 i don't like to change the font size but i tried to add a multiline status on tmux and leaved the second line status as empty, but i was unable to do the same with dracula/tmux though

**Comment by karlbateman** [comment]
I used the following settings to produce a gap between the prompt and the status-bar.

```
set -g status 2
set -g status-format[1] '#[fill=#282A36]'
```

<img width="792" alt="Image" src="https://github.com/user-attachments/assets/51f98311-d3fa-4212-9704-94007db4b3e7" />

**Comment by Theoreticallyhugo** [comment]
i think thats cool! would you like to create a pr detailing this in the docs?

**Comment by Theoreticallyhugo** [comment]
@karlbateman yes, if you have the time to write a nice snippet about it, i think it would be appreciated
